networking-calico/networking_calico/plugins/ml2/drivers/calico/endpoints.py

```diff
@@ -327,6 +327,27 @@ def add_port_qos(self, port, context, port_extra):
         """
         qos = {}
 
+        # Minima, maxima and defaults as specified in the WorkloadEndpoint API,
+        # and implemented for the Kubernetes case in
+        # libcalico-go/lib/backend/k8s/conversion/workload_endpoint_default.go.
+        MINMAX_BANDWIDTH = (1000, 10**15)
+        MINMAX_BW_BURST = (1000, 34359738360)
+        MINMAX_BW_PEAKRATE = (1010, 10**15 + 10**13)
+        MINMAX_BW_MINBURST = (1000, 10**8)
+        DEFAULT_BW_BURST = 4294967296
+
+        MINMAX_PACKET_RATE = (10, 10**12)
+
+        MINMAX_CONNECTIONS = (1, 10**11)
```

Comment:
Same for the `connlimit` module, updated in https://github.com/projectcalico/calico/pull/10489/commits/6964b7421134734382f057fdb94e90b4f61781a5

```suggestion
        MINMAX_CONNECTIONS = (1, 4294967295)
```